Cargo.toml

```
@@ -79,7 +79,7 @@ exclude = ["benches", "distr_test"]
 rand_core = { path = "rand_core" }
 
 [dependencies]
```

I usually put these at the bottom of the file but this should get CI green:

```suggestion
[patch.crates-io.chacha20]
git = "https://github.com/RustCrypto/stream-ciphers"

[dependencies]
```

---

I think it's worth to fix CI before merging. IIRC the patch section does not affect `cargo publish`, so you will be able to remove it in a separate PR after we will release `cahcha20`.